Multisig/UI/ClaimToken/ClaimingAmountViewController/ClaimedAmountInputCell.swift

```diff
@@ -44,6 +46,8 @@ class ClaimedAmountInputCell: UITableViewCell {
         descriptionLabel.text = "Select all tokens or custom amount."
         descriptionLabel.setStyle(.secondary)
 
+        redeemWarningLabel.setStyle(.footnote2.color(UIColor(hex: "#B2BBC0")))
```

Comment:
Should this be its own style? `smallprint` for example?

Reply:
It is used in two places already:
![image](https://user-images.githubusercontent.com/246473/190407749-07032ce7-ce68-4876-b850-8b183c5ea4ab.png)

Reply:
Label style is defined in figma as footnote2 with #B2BBC0 color. Let's revisit this later if we have more usages of this style and color combination. cc @TanyaEfremova